From 22ba958714151bf4f3eb987405bacdb6ed968f77 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Sat, 3 Nov 2018 11:11:54 +0800
Subject: [PATCH 5021/5242] MLK-20224 ARM64: dts: freescale: imx8qxp: disable
 cpu-idle by default

commit  59f54c6d9d145dc64a2d1af4e7c9651f2dbcf831 from
https://source.codeaurora.org/external/imx/linux-imx.git

With cpu-idle enabled, we observe that DSI panel display is NOT
working on i.MX8QXP MEK board, still under debugging, since cpu-idle
ONLY saves ~15mA in runtime, it is NOT valuable enough compare to
whole system, so disable it for now until everything is stable enough.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Nitin Garg <nitin.garg@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8-ca35.dtsi |    4 ----
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi   |    2 --
 2 files changed, 6 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8-ca35.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8-ca35.dtsi
index 2af53ca..2a2f155 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8-ca35.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8-ca35.dtsi
@@ -28,7 +28,6 @@
 			reg = <0x0 0x0>;
 			enable-method = "psci";
 			next-level-cache = <&A35_L2>;
-			cpu-idle-states = <&CPU_SLEEP &CLUSTER_SLEEP>;
 		};
 
 		A35_1: cpu@1 {
@@ -37,7 +36,6 @@
 			reg = <0x0 0x1>;
 			enable-method = "psci";
 			next-level-cache = <&A35_L2>;
-			cpu-idle-states = <&CPU_SLEEP &CLUSTER_SLEEP>;
 		};
 
 		A35_2: cpu@2 {
@@ -46,7 +44,6 @@
 			reg = <0x0 0x2>;
 			enable-method = "psci";
 			next-level-cache = <&A35_L2>;
-			cpu-idle-states = <&CPU_SLEEP &CLUSTER_SLEEP>;
 		};
 
 		A35_3: cpu@3 {
@@ -55,7 +52,6 @@
 			reg = <0x0 0x3>;
 			enable-method = "psci";
 			next-level-cache = <&A35_L2>;
-			cpu-idle-states = <&CPU_SLEEP &CLUSTER_SLEEP>;
 		};
 
 		A35_L2: l2-cache0 {
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index d8db692..b3c7ed3 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -26,7 +26,6 @@
 			reg = <0x0 0x2>;
 			enable-method = "psci";
 			next-level-cache = <&A35_L2>;
-			cpu-idle-states = <&CPU_SLEEP &CLUSTER_SLEEP>;
 		};
 
 		A35_3: cpu@3 {
@@ -35,7 +34,6 @@
 			reg = <0x0 0x3>;
 			enable-method = "psci";
 			next-level-cache = <&A35_L2>;
-			cpu-idle-states = <&CPU_SLEEP &CLUSTER_SLEEP>;
 		};
 	};
 
-- 
1.7.9.5

